packages/wmr/src/wmr-middleware.js

```diff
@@ -101,6 +106,7 @@ export default function wmrMiddleware(options) {
 			pendingChanges.add('/' + filename);
 		} else if (/\.(mjs|[tj]sx?)$/.test(filename)) {
 			if (!moduleGraph.has(filename)) {
+				onChange({ reload: true });
```

Comment:
handles first-load when the module has not entered the graph yet and we fix the error